### DISCUSSION TITLE
[Esame 13 Settembre 2021] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#263] [Esami - Bertini @ Matematica]
![Screenshot 2023-11-14 112656](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/b09956db-983f-4811-9d39-6843752be38e)

**Comment by CasuFrost** [comment]
![image](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/97389762/2ffe97df-bd0f-4277-9614-ec22b8a19312)
